www/xkahj06/DU2-firma.html

```diff
@@ -0,0 +1,58 @@
+﻿<!DOCTYPE html>

```

Comment:
Upozornuji ze veskere kody musi byt naformatovany bud rucne nebo automaticky ve vyjovojem prostredi, jinak to nemuzu schvalit @xhaj06